pkg/resolver/resolver.go

```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/helm/pkg/repo"
 )
 
-// Resolver resolves dependencies from semantic version ranges to a particular version.
+// Resolver resolves libraries/dependencies from semantic version ranges to a particular version.
```

Comment:
Really minor nit here, probably meaningless, but everywhere else that we reference libraries and dependencies at the same time, we use`dependencies/libraries` rather than `libraries/dependencies`. The only reason I could ever see this being relevant is if someone where grepping on `dependencies/libraries` and missed these lines.